src/components/Footer/styles.js

```diff
@@ -0,0 +1,16 @@
+const styles = {
+  buttonContainer: {
+    textDecoration: 'none',
+    flex: 1,
+    display: 'flex',
+    flexFlow: 'column',
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    padding: '0 5%',
+    fontSize: '1em',
+    color: '#FC1F49',
+    height: '90%'
+  }
+}
+
+export default styles
```

Comment:
cool. I haven't ever separated JS styles from the rest of the rendering logic. interested to try this.

Reply:
Yeah, I havent either. I figured in this case since a lot of the buttons in the footer share the same style this would be an easy way to extract common styles